### DISCUSSION TITLE
5G LTE instaed of WIFI

### DISCUSSION THREAD

**Discussion by jeylites** [svpcom/wfb-ng#233] [General]
We use GitHub issues only to discuss WFB bugs and new features. For
questions about using WFB or related components, please use https://github.com/svpcom/wifibroadcast/discussions

Thanks!

Hi, can there be support for 5G LTE instead of WIFI?

**Comment by svpcom** [comment]
Yes. But out don't need for WFB-NG at all in that case. Just install mavlink-router and full-mesh vpn like https://www.tinc-vpn.org/
